### ISSUE TITLE
YouTube playlist is not downloading

### ISSUE THREAD

**Issue by Finont** [ytdl-org/youtube-dl#26248]
## Checklist

- [ x ] I'm reporting a broken site support issue
- [ x ] I've verified that I'm running youtube-dl version **2020.07.28**
- [ x ] I've checked that all provided URLs are alive and playable in a browser
- [ x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x ] I've searched the bugtracker for similar bug reports including closed ones
- [ x ] I've read bugs section in FAQ

## Verbose log

```
y:\Folder1\Net\youtube-dl>youtube-dl -ci --verbose https://www.youtube.com/c/%D0%A1%D1%82%D1%80%D0%BE%D0%B8%D0%BC%D0%B4%D0%BB%D1%8F%D1%81%D0%B5%D0%B1%D1%8F/videos
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-ci', '--verbose', 'https://www.youtube.com/c/%D0%A1%D1%82%D1%80%D0%BE%D0%B8%D0%BC%D0%B4%D0%BB%D1%8F%D1%81%D0%B5%D0%B1%D1%8F/videos']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2020.07.28
[debug] Python version 3.4.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg 3.3.2
[debug] Proxy map: {}
[youtube:user] c: Downloading channel page
WARNING: Unable to download webpage: HTTP Error 404: Not Found
[youtube:user] c: Downloading page #1
ERROR: Unable to download webpage: HTTP Error 404: Not Found (caused by HTTPError()); please report this issue on https://yt-dl.org/bug. Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpq2whuxta\build\youtube_dl\extractor\common.py", line 627, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpq2whuxta\build\youtube_dl\YoutubeDL.py", line 2238, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 502, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 685, in http_error_302
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
```

## Description

youtube-dl does not download video from a specific address: https://www.youtube.com/c/%D0%A1%D1%82%D1%80%D0%BE%D0%B8%D0%BC%D0%B4%D0%BB%D1%8F%D1%81%D0%B5%D0%B1%D1%8F/videos

**Comment by nishanm** [comment]
ive noticed i cant download playlist either. I can download single videos but not when subscribing to a channel or playlist.

**Comment by Finont** [comment]
Issue resolved in version 2020.09.06.
